### PR TITLE
DEVDOCS-4378: [edit] cart line items schema

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-4378](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4378)

## What changed?
Updated `line_items` object.

### CODE REVIEW

**Review by animesh1987 — APPROVED** [review]
Cart changes look good 👍

File: reference/carts.v3.yml [hunk 1]
```diff
@@ -1749,15 +1748,12 @@ components:
                     description: 'Combined download size, in human-readable style. E.g.: `30MB`.'
```
Comment (stanzikratelbc) [inline reference/carts.v3.yml:1748]: Recommend the following for description: 'Specifies the combined download size of all files, in human-readable style, for example: `30MB`.'

File: reference/carts.v3.yml [hunk 2]
```diff
@@ -1791,7 +1787,11 @@ components:
               message:
                 type: string
                 description: Limited to 200 characters.
```
Comment (stanzikratelbc) [inline reference/carts.v3.yml:1789]: Need to add more detail for the description: "The message included in the gift certificate limited to 200 characters."

File: reference/carts.v3.yml [hunk 3]
```diff
@@ -1817,10 +1817,28 @@ components:
               list_price:
                 type: string
                 description: 'Price of the item. With or without tax, depending on your store’s setup. Optional price override.'
```
Comment (stanzikratelbc) [inline reference/carts.v3.yml:1819]: Recommend changing the description to say the following: "Specifies the price of the item. This value can include or exclude tax, depending on the setup of your store."

File: reference/carts.v3.yml [hunk 4]
```diff
@@ -1728,13 +1729,11 @@ components:
                         valueId:
                           type: number
                           description: The product option value identifier.
```
Comment (stanzikratelbc) [inline reference/carts.v3.yml:1731]: I would consider adding, "in number format." and possibly provide an example of a value id

File: reference/carts.v3.yml [hunk 5]
```diff
@@ -1551,16 +1552,16 @@ components:
...
```
Comment (stanzikratelbc) [inline reference/carts.v3.yml:1554]: I would consider adding, "in number format." and possibly provide an example of a value id

**Review by stanzikratelbc — COMMENTED** [review]
Just left some minor comments for changes. Other than that,  LGTM!

**Review by stanzikratelbc — DISMISSED** [review]
Updates look good. Approving this PR. Ready to merge.

**Comment by slsriehl** [comment]
await publication of #955

**Pull request closed without merging**